core-bundle/src/Crawl/Escargot/Subscriber/SearchIndexSubscriber.php

```diff
@@ -67,27 +66,12 @@ public function getName(): string
 
     public function shouldRequest(CrawlUri $crawlUri): string
     {
-        // Check the original crawlUri to see if that one contained nofollow information
-        if (
-            null !== $crawlUri->getFoundOn()
-            && ($originalCrawlUri = $this->escargot->getCrawlUri($crawlUri->getFoundOn()))
-            && $originalCrawlUri->hasTag(RobotsSubscriber::TAG_NOFOLLOW)
-        ) {
+        // Respect robots.txt info and rel="nofollow" attributes
+        if (!Util::isAllowedToFollow($crawlUri, $this->getEscargot())) {
```

Comment:
Should we maybe change line 92 to `$this->getEscargot()` or this one to `$this->escargot` for consistency? ☺️
```suggestion
        if (!Util::isAllowedToFollow($crawlUri, $this->escargot)) {
```